packages/gatsby-plugin-vercel-builder/package.json

```diff
@@ -14,10 +14,10 @@
     "build:src": "tsc -p tsconfig.src.json"
   },
   "dependencies": {
+    "@sinclair/typebox": "0.25.24",
     "@vercel/build-utils": "6.3.2",
     "@vercel/node": "2.9.10",
     "@vercel/routing-utils": "2.1.10",
-    "ajv": "8.12.0",
```

Comment:
I still see ajv being imported in `packages/gatsby-plugin-vercel-builder/src/schemas.ts`

Did you intend to remove it?

Reply:
Removed now: 3608e8a